### DISCUSSION TITLE
“ChatGPT.app” is damaged and can’t be opened. You should eject the disk image.

### DISCUSSION THREAD

**Discussion by khurtwilliams** [lencx/ChatGPT#975] [General]
I downloaded [ChatGPT_1.0.0_macos_aarch64.dmg](https://github.com/lencx/ChatGPT/releases/download/v1.0.0/ChatGPT_1.0.0_macos_aarch64.dmg). I get this error message whenever I try to launch the app. 

> “ChatGPT.app” is damaged and can’t be opened. You should eject the disk image.
>
> This item is on the disk image “ChatGPT_1.0.0_macos_aarch64.dmg”. Safari downloaded this disk image today at 8:57 AM.

**Comment by johnlamox** [comment]
Had the same issue with ChatGPT_1.1.0_macos_aarch64.dmg. Tried "xattr -rc /Applications/chatgpt.app" in Terminal and worked for me.

**Comment by khurtwilliams** [comment]
That works! Thank you.
